### ISSUE TITLE
Using conditions to connect multiple forms.

### ISSUE THREAD

**Issue by paaljoachim** [CalderaWP/Caldera-Forms#1931]
Got a case study here...brainstorming coming.... 

Part 1: The login form.
Only the first two fields are visible.
Register and login.

Clicking register opens a modul to where one adds various information that has been defined. After having filled in various info one clicks send/register and a message is shown in the modul. 
Clicking Login opens two fields below email and password. Filling out both and clicking enter button then part 2 of the form shows up.

Part 2: Signed in one can order member stuff and pay for it.

-----

Bottom line is actually mixing one form with another.

Do login/register form. Then after having clicked send/register the next form shows up.

That means connecting multiple forms together through conditions.

(This would then also use some of the addons such as users and payment.)

**Comment by Shelob9** [comment]
https://calderaforms.com/downloads/caldera-forms-connector/
